### DISCUSSION TITLE
confusion in following the documentation

### DISCUSSION THREAD

**Discussion by chaitanyabsprip** [folke/noice.nvim#277] [Q&A] [answered]
I tried to follow the documentation and even the classes/aliases. I cannot figure out what are valid values for `lsp.documentation.opts.render`. The documentation says its type is `NoiceViewOptions`. Tried to follow it, couldn't figure it out. Can someone help me with this.

**Comment by folke** [comment, marked as answer]
https://github.com/folke/noice.nvim#-views

**Reply by chaitanyabsprip** [reply]
These views are the possible values for `lsp.documentation.opts.render`?

**Reply by folke** [reply]
no, the view options are the options you can use there

**Reply by chaitanyabsprip** [reply]
then I don't understand what `render = "plain"` means?

**Reply by folke** [reply]
that is an option for nvim-notify. If you use another view, like `hover`, then this option is ignored

**Reply by chaitanyabsprip** [reply]
ah, alright makes sense. Thanks!
